elements/rh-datetime/rh-datetime.story.js

```diff
@@ -0,0 +1,191 @@
+import { storiesOf } from "@storybook/polymer";
+import "./rh-datetime";
+
+storiesOf("Datetime", module).add("rh-datetime", () => {
+  const now = new Date();
+  let realtime = now;
+
+  const yearsago = new Date(
+    new Date().setFullYear(new Date().getFullYear() - 10)
+  );
+  const yearago = new Date(
+    new Date().setFullYear(new Date().getFullYear() - 1)
+  );
+  const hoursago = new Date(new Date().setHours(new Date().getHours() - 2));
+  const minutesago = new Date(
+    new Date().setMinutes(new Date().getMinutes() - 10)
+  );
+  const minutesuntil = new Date(
+    new Date().setMinutes(new Date().getMinutes() + 22)
+  );
+  const hoursuntil = new Date(new Date().setHours(new Date().getHours() + 13));
+  const yearuntil = new Date(
+    new Date().setFullYear(new Date().getFullYear() + 1)
+  );
+  const yearsuntil = new Date(
+    new Date().setFullYear(new Date().getFullYear() + 10)
+  );
+
+  function timer() {
+    document.getElementById("realtime").setAttribute("datetime", new Date());
+    window.requestAnimationFrame(timer);
```

Comment:
I think this would be better as a setInterval every second.  As written it's equivalent to a setInterval every 16 milliseconds; all that rAF adds is synchronizing the interval to match the screen's refresh rate.

Reply:
But rAF makes sure that the timer is *super* accurate.

Reply:
It seems like overkill to me but in a story it's not that big a deal.  If it were in the component itself I'd urge setInterval or a throttle within rAF.

Reply:
Ok, let's roll with it.